### PR TITLE
Feat/small fixes

### PR DESCRIPTION
**Pull request by 99mac99**

### CODE REVIEW

File: components/Footer/EndBox.tsx [hunk 1]
```diff
@@ -7,7 +8,7 @@ export default function Footer() {
 		flexDirection: 'row',
```
Comment (adiusz) [inline components/Footer/EndBox.tsx:8]: why not styled-components? :(

File: components/Team/Team.tsx [hunk 2]
```diff
@@ -13,7 +13,7 @@ export default function Team () {
     return (
 			<>
             <Box sx={sx}>
-				<SectionTitle mb={2}> Let's meet us</SectionTitle>
+				<SectionTitle sx={{marginTop: '50px'}}mb={2}> Let's meet us</SectionTitle>
                 <LinearGradient shade={'light'}/>
```
Comment (adiusz) [inline components/Team/Team.tsx:17]: u can use mt prop instead of sx :P

**Review by adiusz — CHANGES_REQUESTED** [review]

File: components/Team/TeamCards.tsx [hunk 3]
```diff
@@ -25,7 +25,7 @@ const TeamCards = () => {
 		fetcher
 	);
 	
-	const sx = { display: 'flex',   width: '60%', flexWrap:'wrap' , heigth:'auto' ,margin: 'auto',   alignItems: 'center',
+	const sx = { display: 'flex', flexWrap:'wrap' , heigth:'auto' , margin: 'auto',   alignItems: 'center',
```
Comment (adiusz) [inline components/Team/TeamCards.tsx:28]: styled-components plzzzz

**Review by adiusz — CHANGES_REQUESTED** [review]

**Comment by adiusz** [comment]
lets be consisent — we agree to use styled-components

File: components/Team/TeamCards.tsx (hunk 3 above)
Comment (99mac99) [reply components/Team/TeamCards.tsx:28]: I'll change it later, we need to push the page xD

**Review by adiusz — APPROVED** [review]

**Pull request merged**